frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js

```diff
@@ -194,4 +201,9 @@ describe('Goals and Objectives', () => {
     expect(screen.getAllByRole('cell')[10]).toHaveTextContent(/program planning and services/i);
     expect(screen.getAllByRole('cell')[11]).toHaveTextContent('1 Objective(s)');
   });
+
+  it('sorts by created on desc when new goals are created', async () => {
+    act(() => renderGoalsAndObjectives([1]));
+    expect(await screen.findByText(/1-1 of 1/i)).toBeVisible();
+  });
```

Comment:
difficult to tell what is being tested here by this assertion

Reply:
I have added a comment and moved relevant call to this test.